packages/core/src/indexing/local-index-api.ts

```diff
@@ -181,6 +209,17 @@ export class LocalIndexApi implements IndexApi {
     // back to the DatabaseIndexApi so that it can populate its internal state.
     // TODO(CDB-2132):  Fix this fragile and circular DatabaseApi initialization
     const modelsToIndex = this.databaseIndexApi.getIndexedModels()
+    for (const model of modelsToIndex) {
+      if (model.indices) {
+        for (const idx of model.indices) {
+          if (!idx.fields) {
+            throw new Error('Fields was undefined')
+          } else if (idx.fields instanceof String) {
```

Comment:
`instanceof String` will return `false` for string literals, is this intentional? Otherwise the check should be `typeof idx.fields === 'string'`

Reply:
Will remove, was debugging something